src/ert/scheduler/driver.py

```diff
@@ -84,25 +84,35 @@ async def _execute_with_retry(
             stdout, stderr = await process.communicate(stdin)
 
             assert process.returncode is not None
+            outputs = (
+                f"exit code {process.returncode}, "
+                f'output: "{stdout.decode(errors="ignore").strip() or "<empty>"}", and '
```

Comment:
why this weird formatting? I'd change make `" "` the outer ones. Also do we need `" "` inside the error message?

Reply:
I think I prefer including quotes (single or double) to wrap the stdout/stderr output to distinguish it from the log from our code.